### DISCUSSION TITLE
Auto-apply scheduler settings and remove apply-button

### DISCUSSION THREAD

**Discussion by fantom-x** [borgbase/vorta#804] [Ideas]
Having apply button on the Schedule action only makes the interface inconsistent: I have tripped over that several times. A suggestion is to either remove it or put it everywhere.

**Comment by m3nu** [comment]
That's true. The reason behind it: Most settings are saved when changed. For the scheduler, we have to restart it, which is a larger operation then just saving something to the DB. Hence the button.

We could probably just auto-apply for every change. Let's see.
